### ISSUE TITLE
Usage of ServiceTasks is tracked via E.T.

### ISSUE THREAD

**Issue by MaxTru** [camunda/camunda-modeler#2218]
> **This is an epic issue which may be broken down into multiple tasks, if needed**

__What should we do?__

We would like to better understand how `bpmn:ServiceTask` elements are used. Hence we would like to track the following metrics via E.T.:

Metric Name | Description | File Type | Sent with Event
-- | -- | -- | --
tasks.serviceTask.count | number of tasks of type serviceTask | BPMN-Camunda, BPMN-Zeebe | diagramOpened, deployment
tasks.serviceTask.implementation.count | number of tasks of type serviceTask which have an implementation | BPMN-Camunda, BPMN-Zeebe | diagramOpened, deployment
tasks.serviceTask.implementation.java | number of tasks of type serviceTask with "Java Class" implementation | BPMN-Camunda | diagramOpened, deployment
tasks.serviceTask.implementation.expression | number of tasks of type serviceTask with "Expression" implementation | BPMN-Camunda | diagramOpened, deployment
tasks.serviceTask.implementation.delegate | number of tasks of type serviceTask with "Delegate Expression" implementation | BPMN-Camunda | diagramOpened, deployment
tasks.serviceTask.implementation.external | number of tasks of type serviceTask with "External" implementation | BPMN-Camunda, BPMN-Zeebe | diagramOpened, deployment
tasks.serviceTask.implementation.connector | number of tasks of type serviceTask with "Connector" implementation | BPMN-Camunda | diagramOpened, deployment

In analogy to `serviceTask`s, this will be embedded as following:
```json
{
  "diagramMetrics": {
    "processVariablesCount": 3,
    "tasks": {
      "userTask": {
        "count": 7,
          "form": {
            "count": 7,
            "embedded": 3,
            "camundaForms": 1,
            "external": 1,
            "generated": 1,
            "other": 1
          },
       "serviceTask": {
        "count": 5,
        "implementation": {
          "count": 5,
          "java": 1,
          "expression": 1,
          "delegate": 2,
          "external": 0,
          "connector": 1
        }
      }
    }
  }
}
```

In particular we need to:

- [x] Track serviceTask counts for zeebe and camunda diagrams
- [ ] Update the [Telemetry documentation](https://github.com/camunda/camunda-modeler/tree/develop/docs/telemetry)

__Why should we do it?__
This will help to better design the serviceTask support in the future of the Camunda Modeler

__Context__
[Telemetry documentation](https://github.com/camunda/camunda-modeler/tree/develop/docs/telemetry)

**Comment by MaxTru** [comment]
For Zeebe I propose:

```json
{
  "diagramMetrics": {
       [...]
       "serviceTask": {
        "count": 5, // count of all serviceTasks
        "implementation": {
          "count": 4 // count of serviceTask which have a taskDefinition provided
        }
      }
    }
  }
}
```

**Comment by MaxTru** [comment]
Decided as described above with @volkergersabeck and @andreasgeier

**Comment by MaxTru** [comment]
Closed by #2224

**Comment by MaxTru** [comment]
Next step: update docs
